app/src/main/java/ca/llamabagel/transpo/ui/trips/TripsViewModel.kt

```diff
@@ -32,12 +42,50 @@ class TripsViewModel @Inject constructor(private val tripsRepository: TripsRepos
             Log.i(TAG, "No stop loaded")
         }
 
-        val response = tripsRepository.getTrips(_stop.value!!.code)
+        apiData = tripsRepository.getTrips(_stop.value!!.code)
 
-        _displayData.value = response.routes.flatMap { route ->
-            route.trips.map { trip -> TripUiModel(route, trip) }
+        when (val copy = apiData) {
+            is Result.Success -> {
+                _displayData.value = copy.data.routes.flatMap { route ->
+                    route.trips.map { trip -> TripUiModel(route, trip) }
+                }
+                    .sortedBy { it.trip.adjustedScheduleTime }
+                    .map(::TripItem)
+                updateViewerData()
+            }
+            is Result.Error -> {
+                // TODO: Handle errors
```

Comment:
This TODO intentional?

Reply:
For now yeah, not sure how we're going to handle error states.